src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -524,8 +521,37 @@ TEST_F(HMICapabilitiesTest,
 
   EXPECT_TRUE(phone_capability_so.keyExists("dialNumberEnabled"));
   EXPECT_TRUE(phone_capability_so["dialNumberEnabled"].asBool());
+}
+
+TEST_F(
+    HMICapabilitiesTest,
+    LoadCapabilitiesFromFile_CheckPhoneCallNavigation_SuccessLoadAndConvert) {
+  const std::string content_to_save =
+      "{\"UI\": "
+      "{\"hmiCapabilities\":\
+       {\"navigation\":true,\"phoneCall\":true,\"videoStreaming\":true}}}";
+
+  CreateFile(kHmiCapabilitiesCacheFile);
+  const std::vector<uint8_t> binary_data_to_save(content_to_save.begin(),
+                                                 content_to_save.end());
+  file_system::Write(kHmiCapabilitiesCacheFile, binary_data_to_save);
+  hmi_capabilities_->Init(last_state_wrapper_);
+
+  const auto ui_hmi_capability_so = *(hmi_capabilities_->ui_hmi_capabilities());
+  EXPECT_TRUE(ui_hmi_capability_so.keyExists("navigation"));
+  EXPECT_TRUE(ui_hmi_capability_so.keyExists("phoneCall"));
+  EXPECT_TRUE(ui_hmi_capability_so.keyExists("videoStreaming"));
+
+  hmi_capabilities_->set_phone_call_supported(
+      ui_hmi_capability_so["phoneCall"].asBool());
+  hmi_capabilities_->set_navigation_supported(
+      ui_hmi_capability_so["navigation"].asBool());
+  hmi_capabilities_->set_navigation_supported(
+      ui_hmi_capability_so["videoStreaming"].asBool());
```

Comment:
@RomanReznichenkoLuxoft why are you changing these fields? I believe the same code will be executed in `Init()` while parsing UI capabilities object

Reply:
done